Sources/Subscription/Services/SubscriptionService.swift

```diff
@@ -101,4 +82,19 @@ public struct SubscriptionService: APIService {
         public let customerPortalUrl: String
     }
 
+    // MARK: -
+
+    public static func confirmPurchase(accessToken: String, signature: String) async -> Result<ConfirmPurchaseResponse, APIServiceError> {
+        let headers = makeAuthorizationHeader(for: accessToken)
+        let bodyDict = ["signedTransactionInfo": signature]
+
+        guard let bodyData = try? JSONEncoder().encode(bodyDict) else { return .failure(.encodingError) }
+        return await executeAPICall(method: "POST", endpoint: "purchase/confirm/apple", headers: headers, body: bodyData)
```

Comment:
Maybe move the strings in this method to constants?

Reply:
Makes sense but I am considering move the implementation to use `Networking` package.